src/affexpr.jl

```diff
@@ -125,6 +125,20 @@ function assert_isfinite(a::AffExpr)
     end
 end
 
+"""
+    resultvalue(v::AffExpr)
+
+Evaluate an `AffExpr` given the result returned by a solver.
+Replaces `getvalue` for most use cases.
+"""
+function resultvalue(a::AffExpr)
```

Comment:
Could you write a generic version that works for any ``VariableToValueMap`` and then have this as a single-line special case?